tests/integration/downstream/downstream_test_case.py

```diff
@@ -0,0 +1,38 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+from app.submitter.submitter import SubmitterFactory, Submitter
+from app.submitter.converter import Converter
+
+
+class MyMockSubmitter(Submitter):
+    def __init__(self):
+        self._message = None
+        self._submitted_at = None
```

Comment:
Is there a reuse for self._submitted_at? It doesn't appear to be used anywhere, apart from in the teardown method where it's just set back to None?